[DOCS-10907] CCM Custom Recommendations (#29509)
DataDog/documentation: content/en/cloud_cost_management/recommendations/custom_recommendations.md

---
title: Custom Recommendations
description: Learn how to customize and manage custom recommendations to fit your business needs.
further_reading:
- link: "/cloud_cost_management/"
  tag: "Documentation"
  text: "Cloud Cost Management"
- link: "/cloud_cost_management/recommendations"
  tag: "Documentation"
  text: "Cloud Cost Recommendations"

---

{{< callout url="#" btn_hidden="true" header="false">}}
  Custom Recommendations is in Preview.
{{< /callout >}}

## Overview

{{< img src="cloud_cost/.png" alt="Configure custom recommendations from the Cloud Cost Settings page." style="width:80%;" >}}

> While [CCM Recommendations][1] are designed to be informative and actionable from the start, you can customize recommendations that you receive to fit your specific business use cases and needs. Set your own metric thresholds and evaluation time frames to reduce unnecessary alerts and focus on the recommendations that matter most.

Reviewer: I think the "While" makes it sound like these two are in competition, but is this more of an additional feature to CCM Recommendations?

```suggestion
[CCM Recommendations][1] are designed to be informative and actionable from the start. You can customize these recommendations to fit your specific business use cases and needs. Set your own metric thresholds and evaluation time frames to reduce unnecessary alerts and focus on the recommendations that matter most.
```

With custom recommendations, you can:
- Configure multiple rules for any default recommendation
- View the history and changes made to your configurations
- Modify, restore, or delete previously set rules

## Customize a recommendation

<div class="alert alert-info">Customizations reflect within 24 hours, when recommendations are generated next.</div>

To access custom recommendations, go to [**Cloud Cost > Settings > Configure Recommendations**][2].

On this page, you can see a list of out-of-the-box recommendations that can be customized.

Click a recommendation, then click **Create New Configuration** to get started.

### Step 1: Set custom metric thresholds

> You can determine a value for each metric threshold used to generate recommendations for resources.

Reviewer: ```suggestion
You can set a value for each metric threshold used to generate resource recommendations.
```

> Any metric threshold unavailable for customization is grayed out.

Reviewer: ```suggestion
Metric thresholds that cannot be customized appear grayed out.
```

### Step 2: Customize the evaluation time frame

> Adjust the evaluation time frame to better match your business's seasonality or operational patterns. Options include: 1 week, 2 weeks, 15 days, 3 weeks, 1 month, 2 months, and 3 months.

Reviewer: ```suggestion
Adjust the evaluation time frame to match your business's seasonality or operational patterns. Options include: 1 week, 2 weeks, 15 days, 3 weeks, 1 month, 2 months, and 3 months.
```
